### PR TITLE
Adds inertia.js adapter

### PR DESCRIPTION
**Pull request by ishanvyas22**
Adds https://github.com/ishanvyas22/cakephp-inertiajs

Addressed all the changes from #327

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,6 +263,7 @@ Additional lists you might find useful:
 - :strawberry: [Feedback plugin](https://github.com/dereuromark/cakephp-feedback) - Allow visitors to send quick and easy feedback incl. a screenshot via sidebar form.
 - :strawberry: [Flash plugin](https://github.com/dereuromark/cakephp-flash) - More powerful flash messages for your application.
 - [Heartbeat plugin](https://github.com/orca-services/cakephp-heartbeat/) - Monitor the hearbeat of your application (e.g. whether the database is available and up-to-date).
+- [Inertia.js adapter](https://github.com/ishanvyas22/cakephp-inertiajs) - The CakePHP adapter for Inertia.js.
```
Comment (dereuromark) [inline README.md:266]:
```suggestion
- [Inertia plugin](https://github.com/ishanvyas22/cakephp-inertiajs) - Server side adapter for Inertia.js.
```

**Comment by dereuromark** [comment]
Shouldnt the list now only accept cake 4 compatible ones? I am not sure what others think here.

**Comment by ishanvyas22** [comment]
Hi @dereuromark, updating to Cake4 is in my todo list. So yes, this plugin will soon be compatible with Cake4 too.

**Comment by ishanvyas22** [comment]
Plugin now supports Cake 4.x 🎉 

https://github.com/ishanvyas22/cakephp-inertiajs/releases/tag/2.0.0

**Pull request merged**